analyzers.ruleset

```diff
@@ -228,7 +228,7 @@
     <Rule Id="SA1620" Action="None" />
     <Rule Id="SA1621" Action="None" />
     <Rule Id="SA1622" Action="None" />
-    <Rule Id="SA1623" Action="Warning" />
+    <Rule Id="SA1623" Action="None" />
```

Comment:
This is annoying, keeps asking you to start your comments with gets or sets even if the property is boolean